packages/sdk-codegen/src/python.gen.ts

```diff
@@ -263,7 +274,27 @@ ${this.hooks.join('\n')}
   }
 
   initArg(indent: string, property: IProperty) {
-    return `${indent}self.${property.name} = ${property.name}`
+    let propName = property.name
+    if (this.pythonKeywords.has(propName)) {
+      propName = propName + '_'
+    }
+    return `${indent}self.${propName} = ${propName}`
+  }
+
+  typeProperties(type: IType) {
+    return Object.values(type.requiredProperties).concat(
+      Object.values(type.optionalProperties)
+    )
+  }
+
+  private isBareForwardRef = (prop: IProperty) => {
+    // contains a "bare" forward reference e.g. `result_format: "ResultFormat"`
+    // so we need to emit the `__annotations__` property
+    return (
+      prop.required &&
+      (prop.type.customType || prop.type instanceof EnumType) &&
```

Comment:
Oh, EnumType should definitely have a truthy `customType`. If it's not, we should get around to fixing that.

Reply:
I think it doesn't which is why I had to add the EnumType check in here.